src/modules/nfts/components/wizard/InfoScreen.tsx

```diff
@@ -113,6 +114,8 @@ export default function InfoScreen({
   dispatch,
   currentImage,
 }: Props) {
+  const [errors, setErrors] = useState<string[]>([]);
```

Comment:
Looks like there is a mix of using the validation engine from antd and this. Why wasn't a form.item validation sufficent?

Reply:
Aha, found the correct documentation for custom validators now, will implement that.

Reply:
sweet. ty.

Reply:
@kespinola I switched to the ant D custom validator (works great!), but had to keep the local error state. Otherwise it threw an error on every new attribute added (including the first one), instead of only validating on clicking Next.